coil-video/api/coil-video.api

```diff
@@ -4,6 +4,7 @@ public abstract class coil/fetch/VideoFrameFetcher : coil/fetch/Fetcher {
 	public static final field VIDEO_FRAME_OPTION_KEY Ljava/lang/String;
 	public fun <init> (Landroid/content/Context;)V
 	public fun fetch (Lcoil/bitmap/BitmapPool;Ljava/lang/Object;Lcoil/size/Size;Lcoil/decode/Options;Lkotlin/coroutines/Continuation;)Ljava/lang/Object;
+	public static final synthetic fun fetch$suspendImpl (Lcoil/fetch/VideoFrameFetcher;Lcoil/bitmap/BitmapPool;Ljava/lang/Object;Lcoil/size/Size;Lcoil/decode/Options;Lkotlin/coroutines/Continuation;)Ljava/lang/Object;
```

Comment:
It's a bug in `1.4.31` that this is declared public, but it can't be called by Java and it's a Kotlin implementation detail so I wouldn't consider it part of the public API.